.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: ruby
+rvm:
+  - "jruby"
```

Comment:
You don't need the quotes around the value. Also, do we not want to specify a version?

e.g.

```yaml
rvm:
  - jruby
  - 1.7.17
```